src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletController.java

```diff
@@ -66,4 +66,13 @@ public ResponseEntity<WalletDTO> deleteWalletById(@Valid @Min(1) @PathVariable L
 
         return new ResponseEntity<>(headers, HttpStatus.OK);
     }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<WalletDTO> findWalletById(@Valid @Min(1) @PathVariable Long id) {
```

Comment:
Please remove @Valid annotation and add @NotNull instead in every methods in this class where argument is not complex object, i.e.: Long

Please add @Validated annotation on his class - because on methods:  updateWallet, createWallet argument is complex -  `UpdateWalletDTO`, `CreateWalletDTO`.

Reply:
Done